lib/parse/index.js

```diff
@@ -56,18 +56,31 @@ module.exports = function(css, options){
    * Error `msg`.
    */
 
+  var errorsList = [];
+
   function error(msg) {
+    var errorObj;
+    var fullErrorMsg = options.source + ':' + lineno + ':' + column + ': ' + msg;
+
     if (options.silent === true) {
-      return false;
+      errorObj = {
+        message: fullErrorMsg
+      };
+    } else {
+      errorObj = new Error(fullErrorMsg);
```

Comment:
It wouldn't stringify correctly, as the `message` property of Error is non enumerable.

```js
var err = new Error('a message');
console.log(JSON.stringify(err));
// {}
```